pkg/datastore/database.go

```diff
@@ -112,7 +112,7 @@ func (db *relationalDb) getDBTransaction(ctx context.Context, tableName string,
 			TABLE_NAME:   tableName,
 			"authorizer": TypeName(db.authorizer),
 		})
-		db.logger.Error(err)
+		db.logger.Debug(err)
```

Comment:
Why change logging level to Debug? Isn't this an actual error that needs to be logged at Error level?

Reply:
Never log the bare error without context, it's useless, consumer can log the error if its a failure case.